### PR TITLE
Creating views

### PR DESCRIPTION
**Pull request by mouseorcleg**

### CODE REVIEW

**Comment by mouseorcleg** [comment]
First glance:

Screenshot 2023-01-06 at 16 36 34

**Comment by mouseorcleg** [comment]
AddView basics:

Screenshot 2023-01-06 at 16 58 05

**Comment by mouseorcleg** [comment]
Updated AddView:

Screenshot 2023-01-06 at 18 19 33

**Comment by mouseorcleg** [comment]
Maybe I will rework TypePicker later, but it's done for now.

**Comment by mouseorcleg** [comment]
The final-final look
Screenshot 2023-01-06 at 19 34 00

File: TomatoTomato2/Views/AddView.swift [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+import SwiftUI
+
+struct AddView: View {
+    
+    @State var textFieldText: String = ""
+    @State var sizePickerSelection: String = "L"
+    @State var typePickerSelection: String = "Research"
+    
+    var sizePickerOptions: [String] = ["XS", "S", "M", "L", "XL"]
```
Comment (togramago) [inline TomatoTomato2/Views/AddView.swift:16]: please move outside of VIew. 
even sth like this: `let sizes = ["XS", "S", "M", "L", "XL"]` on line 11, and `var sizePickerOptions: [String] = sizes`

**Pull request merged**